web-common/src/components/menu/DashboardMetricsDraggableList.svelte

```diff
@@ -208,50 +264,92 @@
       {#if selectedItems.length < allItems.length}
         <span class="flex-none h-px bg-slate-200 w-full" />
         <div class="hidden-section flex flex-col flex-1 min-h-0 p-1.5 pt-0">
-          <DraggableList
-            items={hiddenDraggableItems}
-            bind:searchValue={searchText}
-            minHeight="auto"
-            maxHeight="200px"
-            onItemClick={handleHiddenItemClick}
+          <!-- Hidden items header -->
+          <div
+            class="flex-none flex py-1.5 justify-between px-2 sticky top-0 from-popover from-80% to-transparent bg-gradient-to-b"
           >
```

Comment:
This PR shouldn't remove this instance of our `DraggableList` component. Instead, any changes should be made to the `DraggableList` component directly.

Otherwise, we'll effectively be maintaining two different "draggable lists" and they'll easily diverge over time.